src/interfaces/checkboxes/input.vue

```diff
@@ -65,16 +124,100 @@ export default {
       }
 
       this.$emit("input", selection);
+    },
+
+    saveSort() {
+      let selection = this.selection;
+      let staged = _.map(this.sortableList, function(k) {
+        return k.val ? k.val : k;
+      });
+      staged = staged.filter(val => selection.includes(val));
+      return this.$emit("input", staged);
+    },
+
+    trimValues(arr, comp) {
+      return arr
+        .map(e => e[comp])
+        .map((e, i, final) => final.indexOf(e) === i && i) // store the keys of the unique objects
+        .filter(e => arr[e]) // eliminate the dead keys & store unique objects
+        .map(e => arr[e]);
+    },
+
+    findLabel(choice, k) {
+      if (choice.find(x => x.val === k)) {
+        // find labels in in interface options
+        if (choice.find(x => x.val === k).label) {
+          return choice.find(x => x.val === k).label;
+        } else {
+          return choice.find(x => x.val === k);
+        }
+      } else if (k !== null) {
+        // return key if no label was found
+        return k;
+      }
     }
   }
 };
 </script>
 
 <style lang="scss" scoped>
 .interface-checkboxes {
-  max-width: var(--width-x-large);
+  width: var(--width-x-large);
+  max-width: 100%;
   display: grid;
   grid-gap: 20px;
-  grid-template-columns: repeat(4, 1fr);
+  grid-template-columns: repeat(1, 1fr);
+
+  @media only screen and (min-width: 330px) {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @media only screen and (min-width: 480px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+  @media only screen and (min-width: 800px) {
+    grid-template-columns: repeat(4, 1fr);
+  }
+
+  &.single {
+    grid-template-columns: repeat(1, 1fr);
+  }
+  &.draggable {
+    max-width: 100%;
+    .sortable-box {
+      position: relative;
+      transition: opacity ease-in-out 0.2s, background-color ease-in-out 0.3s;
```

Comment:
Use variables for transitions

```suggestion
      transition: opacity var(--transition) var(--medium), background-color var(--transition) var(--slow);
```

(medium === 0.2s, slow === 0.3s)